style.css

```diff
@@ -1 +1,357 @@
+  /* header grid */
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  height: 200px;
+  margin-bottom: 0;
+}
+
+.logo {
+  grid-area: 1 / 1 / 3 / 2;
+  background-image: #8be5f3;
+}
+.logo img {
+  width: 200px;
+  height: 200px;
+}
+
+header p {
+  display: flex;
+  justify-content: center;
+  font-family: "Cookie", sans-serif;
+  font-size: 20px;
+  margin: 10px;
+}
+/* header navigation */
+nav {
+  grid-area: 1 / 2 / 2 / 3;
+}
+
+header ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+header ul li {
+  margin: 10px;
+}
+
+/* menu */
+.navigation {
+  display: none;
+}
+.menu {
+  border: 1px solid var(--green);
+  padding: 1rem;
+  position: fixed;
+  z-index: 2;
+  background-color: var(--red-light);
+  opacity: 0.9;
+  top: -800px;
+  right: 70px;
+  transition: all 0.3s ease-in-out;
+  border-radius: 5px;
+}
+header ul li a {
+  text-decoration: none;
+  color: var(--black);
+}
+
+#check {
+  display: none;
+}
+
+/* burger menu */
+.open-menu {
+  top: 5%;
+  right: 20px;
+  transform: translateY(-50%);
+  position: absolute;
+  cursor: pointer;
+  transition: all 0.4s ease;
+}
+
+#check:checked ~ .open-menu {
+  transform: rotate(-90deg);
+}
+#check:checked ~ .menu {
+  top: 0;
+}
+/* header text */
+header p {
+  grid-area: 2 / 2 / 3 / 3;
+}
+
+/* main */
+main {
+  grid-area: 2 / 1 / 3 / 2;
+  /* main grid */
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  background-color: var(--pink);
+  height: fit-content;
+}
+.container {
+  grid-area: 1 / 1 / 2 / 2;
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.container h1 {
+  font-size: 42px;
+  font-family: "Cookie", sans-serif;
+}
+
+.container p {
+  font-size: 30px;
+}
+
+.mainpicture {
+  grid-area: 2 / 1 / 3 / 2;
+  background-image: url(/image/Anniversary-Cake-Birthday-60th-Diamond-1024x731.jpeg);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 10px;
+  box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
+    rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+}
+
+.pic {
+  grid-area: 3 / 1 / 4 / 2;
+  /* collage grid */
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: auto;
+  grid-column-gap: 10px;
+  grid-row-gap: 10px;
+  margin: 10px;
+  height: fit-content;
+}
+
+.pic img {
+  display: block;
+  width: 100%;
+  height: 100%;
+  box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
+    rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+}
+
+/* footer */
+footer {
+  grid-area: 3 / 1 / 4 / 2;
+  background-color: var(--yellow-light);
+  /* footer grid */
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  height: fit-content;
```

Comment:
I don't think this property is needed, the footer will naturally grow to fit its content